sdks/python/apache_beam/examples/inference/pytorch_image_classification.py

```diff
@@ -114,10 +115,13 @@ def run(argv=None, model_class=None, model_params=None, save_main_session=True):
     model_class = MobileNetV2
     model_params = {'num_classes': 1000}
 
-  model_loader = PytorchModelLoader(
-      state_dict_path=known_args.model_state_dict_path,
-      model_class=model_class,
-      model_params=model_params)
+  # the input to RunInference transform is keyed. Wrap
+  # PytorchModelHandler on KeyedModelHandler for keyed examples.
+  model_loader = KeyedModelHandler(
+      PytorchModelHandler(
+          state_dict_path=known_args.model_state_dict_path,
+          model_class=model_class,
+          model_params=model_params))
```

Comment:
Do we still need the type hints on line 138?

Reply:
yes, still getting the error(expecting Union) if I remove the output hint.

Reply:
Do you think it's because `RunInference` is imported from `api.py` instead of `base.py` and the `RunInference` in `api.py` has `Union[...]` type?